### PR TITLE
VTCCA-3166: added client property links endpoint

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala [hunk 1]
```diff
@@ -102,6 +103,27 @@ class ExternalPropertyLinkApi @Inject()(
           ).flatten
       )
 
+  def getAssignedPropertyLinksForClient(
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/voapropertylinking/connectors/modernised/ExternalPropertyLinkApi.scala:106]: `getClientPropertyLinks` is more consistent

File: app/uk/gov/hmrc/voapropertylinking/controllers/PropertyLinkingController.scala [hunk 2]
```diff
@@ -148,6 +149,15 @@ class PropertyLinkingController @Inject()(
       .fold(NotFound("clients property links not found"))(propertyLinks => Ok(Json.toJson(propertyLinks)))
   }
 
+  def getAssignedPropertyLinksForClient(
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/voapropertylinking/controllers/PropertyLinkingController.scala:152]: `getClientPropertyLinks` is more consistent

**Comment by darrell-plant** [comment]
LGTM 👍 with a few comments about naming

**Comment by reddy-yattapu** [comment]
👍

**Pull request merged**